data/projects/enhancing_coala_quickstart.md

```diff
@@ -0,0 +1,37 @@
+coala-quickstart is a tool that helps users to quickly get started with coala. It generates a `.coafile` (coala's configuration file) that is tailored to the project. Currently, coala-quickstart automatically identifies relevant bears for the project based on the languages used and has a clean and simple command-line interface for taking user’s input when required.
```

Comment:
Limit to 80 chars or something like that - not really enforced in this repo, but it's easier to view diffs later on.